.github/workflows/tests.yml

```
@@ -18,116 +18,93 @@ on:
 jobs:
   phpunit:
     runs-on: ubuntu-20.04
-    name: PHPUnit tests
+    name: PHPUnit ${{ matrix.php_version }} ${{ matrix.mautic_version }}
+    strategy:
+      matrix:
+        mautic_version: ["5.x-dev", "4.x-dev"]
+        php_version: ["7.4", "8.0", "8.1"]
```

As per https://github.com/mautic/api-library/pull/286 PHP 7 is no longer supported.

```suggestion
        php_version: ["8.0", "8.1"]
```